### ISSUE TITLE
Ahn'Kahet The Old Kingdom : Boss Amanitar

### ISSUE THREAD

**Issue by Stealthmate** [dalaranwow/dalaran-wow#1894]
This is the optional boss who only appears on Heroic difficulty (and who almost nobody kills). He is required for the achievement (http://wotlk.wowhead.com/achievement=492).

**How it should work:** All of this can be read here: http://www.wowwiki.com/Amanitar.
About the problem - when someone has the Mini debuff (http://www.wowhead.com/spell=57055), they have to personally kill a Healthy Mushroom (http://www.wowhead.com/npc=30391) to remove the debuff. Also, if the player does NOT have the debuff, when killing the mushroom the Player and everyone nearby will gain the buff Potent Fungus (http://www.wowhead.com/spell=56648)

**How it actually works:** The problem MAY be only "visual" (similar to https://github.com/dalaranwow/dalaran-wow/issues/1255, S&B procs, but Shield Slam ignores it), but I haven't tested to see if it is. 
Basically, once you get the debuff, you cannot lose it, even if you kill healthy mushrooms. Now I said it may be visual, because I noticed that one time I DID get the Potent Fungus buff, which should be available ONLY if you are NOT debuffed with Mini. I would ask someone to confirm this. 

Otherwise, the mushrooms do spawn, both healthy and poisonous, although I do not know if the poisonous ones deal damage. Again, someone please confirm this.

**Comment by codyflood** [comment]
I'd like to bump this issue as it still seems to exist, whether it is purely visual or the mini effect persists I have yet to fully be able to test

**Comment by Rushor** [comment]
http://wotlk.openwow.com/npc=30258

**Comment by Rushor** [comment]
Scripts/Ahnkahet: Boss Amanitar - Add several removes for debuff "Mini" on death, reset of the boss and on kill of the Healthy Mushrooms

* also add the buff "Potent Fungus" on death of Healthy Mushrooms
* closes https://github.com/dalaranwow/dalaran-wow/issues/1894
* commit: 6eaf38a851809ad96e0237dc2e9a0cf935d5ec3f

![GitHub Logo](http://i.imgur.com/qOAAOFK.jpg)
![GitHub Logo](http://i.imgur.com/EX1qmZ9.jpg)
![GitHub Logo](http://i.imgur.com/J5YNsDK.jpg)

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
